pontoon/teams/views.py

```diff
@@ -276,9 +276,12 @@ def request_item(request, locale=None):
             body=mail_body,
             from_email=settings.LOCALE_REQUEST_FROM_EMAIL,
             to=settings.PROJECT_MANAGERS,
-            cc=locale.managers_group.user_set.exclude(pk=user.pk).values_list(
-                "email", flat=True
+            cc=list(
```

Comment:
It's possible that the person that makes the request is also a locale manager, in which case we'll list their email address twice. Let's prevent this by wrapping the entire value of `cc` in `set()`.